config/develop/bridgeworker.yaml

```diff
@@ -1,9 +1,9 @@
 template_path: bridgeworker.yaml
 stack_name: bridgeworker-develop
 parameters:
-  AwsDefaultVpcId: vpc-9c70bbf9
+  AwsDefaultVpcId: vpc-f389e98a
   AwsSnsNotificationEndpoint: bridgeworker-develop@sagebase.org
-  BackfillBucket: org-sagebridge-backfill-develop
+  BackfillBucket: org-sagebridge-backfill-devdevelop
```

Comment:
This is because S3 bucket names are globally unique. I didn't want to delete the stacks in the Prod AWS account before we create the stacks in the Dev AWS account, so I had to create the buckets with new names to avoid name collision. I went with devdevelop and devstaging.